Smartling.API/Model/TranslationJob.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Newtonsoft.Json;
+using System;
+
+namespace Smartling.Api.Model
+{
+    public class TranslationJob
```

Comment:
Seems like we already have Job model: https://github.com/Smartling/api-sdk-net/blob/master/Smartling.API/Model/Job.cs

How are they different?

Reply:
right, duplicate

Reply:
Removed